ext/afform/core/CRM/Afform/AfformScanner.php

```diff
@@ -149,6 +149,8 @@ public function getMeta($name) {
       'is_dashlet' => FALSE,
       'is_public' => FALSE,
       'is_token' => FALSE,
+      'authx_timeout' => 0,
```

Comment:
This defaults to `0` but in `Civi\Afform\Tokens` it uses the `??` operator to check if this is set. So I think it should default to `NULL`.

Reply:
Thanks, fixed